test/config-files/comprehensive-config

```diff
@@ -34,18 +34,18 @@ copy-in-before   = []
 copy-out-after   = []
 copy-timeout     = 0
 image            = ["centos/7:2004.01"]
-# image_backend    = None
-# image_frontend   = None
-# name             = None
+image_backend    = "$HOME/.local/share/transient/backend"
```

Comment:
Does the variable expansion here actually work?

Reply:
Oh, I see you opened #92. If we're going to do that, then this PR is blocked until we add the variable expansion feature.

Reply:
It does not, but I opened https://github.com/ALSchwalm/transient/issues/92 because of this.

Maybe the best solution would be for me to change the path and add a comment that indicates variable expansion is not currently supported. Then we can remove it after we add that feature.

Reply:
Added commit that supports environment variable expansion and mentioned the fix in the PR body.